src/scraper/scraper.cpp

```diff
@@ -51,6 +51,8 @@ struct ScraperFileManifestEntry
     uint256 hash; // hash of file
     int64_t timestamp;
     bool current;
+    bool excludefromcsmanifest;
+    std::string filetype;
```

Comment:
Would it be better to use an enum for these types?

Reply:
No. Not in this case, and I considered doing an enum here. But the juice in this instance is not worth the squeeze. Remember the entries are written out to a manifest file, which I want to be human readable. Using an enum simply makes the lookup more complicated to transform to a string on the write out to the file, or conversely for input to memory. Further if you record the enum position rather than the string to avoid the transformation on serialization, the value is brittle and the order of the enumeration becomes important if additional categories are added later and you want to retain upgradability from an older file.

The usual guidance about using enum for categorized set of constants is not the overriding factor here, and the scraper itself is the only consumer of that structure anyway.

Reply:
Well said. I forgot the bit about outputting it to the file.

Reply:
I wouldn't have bothered, but I wanted the scraper to be able to pick up where it left off if the scraper is shutdown and restarted...